.github/workflows/release.yml

```
@@ -0,0 +1,616 @@
+          
+          # This would typically use tools like api-extractor or similar
+          # For now, we'll check commit messages
+          LAST_TAG=$(git describe --tags --abbrev=0 2>/dev/null || echo "")
+          
+          if [ -n "$LAST_TAG" ]; then
+            BREAKING_CHANGES=$(git log $LAST_TAG..HEAD --grep="BREAKING CHANGE" --oneline || echo "")
+            
+            if [ -n "$BREAKING_CHANGES" ]; then
+              echo "⚠�� Breaking changes detected:"
+              echo "$BREAKING_CHANGES"
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)